ui/app/components/ui/mascot/mascot.component.js

```diff
@@ -46,6 +69,32 @@ export default class Mascot extends Component {
 
   componentDidMount () {
     this.mascotContainer.current.appendChild(this.logo.container)
+    this.directionTargetMap = directionTargetGenerator(this.mascotContainer.current.getBoundingClientRect())
+
+    const { lookAtTarget, lookAtDirection } = this.props
+
+    if (lookAtTarget?.x && lookAtTarget?.y) {
+      this.logo.lookAtAndRender(lookAtTarget)
+    }
+    if (lookAtDirection) {
+      this.logo.lookAtAndRender(this.directionTargetMap[lookAtDirection])
+    }
```

Comment:
Why shouldn't this be an if/if rather than an if/else?

Reply:
You're right, it should be if/else. Fixed in 4ff2b1138